lib/src/draw_config_context.dart

```diff
@@ -329,29 +322,25 @@ class DRAWConfigContext {
       (_fetchOptional(key) ?? _fetchDefault(key) ?? kNotSet);
 
   /// Returns path to user level configuration file.
+  /// Special Behaviour: if User Config Enviroment var unset, uses [$HOME] or ['/']
   Uri _getUserConfigPath() {
     final environment = Platform.environment;
     String osConfigPath;
     // Load correct path for user level configuration paths based on operating system.
     if (Platform.isMacOS) {
       osConfigPath = path.join(environment[kMacEnvVar], '.config');
     } else if (Platform.isLinux) {
-      osConfigPath = environment[kLinuxEnvVar];
+      osConfigPath = environment[kLinuxEnvVar] ??
+          path.join(environment[kLinuxHomeEnvVar], '.config');
     } else if (Platform.isWindows) {
       osConfigPath = environment[kWindowsEnvVar];
     } else {
       throw new DRAWInternalError('OS not Recognized by DRAW');
     }
+    osConfigPath ??= '/';
```

Comment:
Have you tried this on Windows? I have a feeling it might not like '/' as a root directory (it probably won't be good to just use C:\ either, since you really don't have to use C:\ as your root drive). I know, Windows sucks 👎